libkbfs/cr_chains.go

```diff
@@ -1133,6 +1134,50 @@ func (ccs *crChains) findPathForDeleted(mostRecent BlockPointer) path {
 	}
 }
 
+func (ccs *crChains) findPathForCreated(mostRecent BlockPointer) path {
+	// Find the parent chain that deleted this one.
+	for ptr, chain := range ccs.byMostRecent {
+		for _, op := range chain.ops {
+			co, ok := op.(*createOp)
+			if !ok {
+				continue
+			}
+			for _, ref := range co.Refs() {
+				if ref == mostRecent {
+					// If the path isn't set yet, recurse.
+					p := co.getFinalPath()
```

Comment:
is "final path" the parent path of the entry here?

Reply:
Yep, exactly!